README.md

```diff
@@ -0,0 +1,32 @@
+#  Tkey ios example application document
+
+After building complete, you can login via your google account.
+Clicking this button will make account for you, using customAuth sdk.
+
+Once you're logged in, you can run a number of tkey-related functions.
+Buttons allow you to test various modules and tkey built-in functions.
+
+## Main Page
+![image](mainPage.png)
+
+### how to start/Users/janghyeong-gyu/workspace/web3auth/tkey-rust-ios-example/mainPage.png
+
+Once you have the final tkey from initialize and reconstruct tkey, you can test all the features.
+The first time you run Initialize and reconstruct tkey, two shares will be created and the threshold will be set to two.
+This means that both shares will be required for login. (2/2 setting)
```

Comment:
Only true for new accounts, existing accounts would need to have the saved shares.